modules/gdscript/gdscript_editor.cpp

```diff
@@ -427,7 +428,30 @@ void GDScriptLanguage::debug_get_globals(List<String> *p_globals, List<Variant>
 }
 
 String GDScriptLanguage::debug_parse_stack_level_expression(int p_level, const String &p_expression, int p_max_subitems, int p_max_depth) {
-	return "";
+    List<String> names;
+	List<Variant> values;
+	debug_get_stack_level_locals(p_level, &names, &values, p_max_subitems, p_max_depth);
```

Comment:
The indentation in this code is invalid, please make sure to only use tabs for indentation. You can run clang-format when committing code by setting up `pre-commit` with:
```
pip install pre-commit
pre-commit install
```